proj.4/build.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
 ./configure --prefix=$PREFIX --without-jni
+
 make
+
+if [[ $(uname) == Linux ]]; then
+    make check
+fi
```

Comment:
Yep.  I mixed the logs here.  Let me see if I can find it.  (Or better yet, remove the conditional and see if it fails again.)